unicorecmsmama/templates/content.pt

```diff
@@ -7,6 +7,9 @@
             <div class="h1" tal:condition="primary_category">
                 <a href="/content/list/${primary_category.uuid}/">${primary_category.title}</a>
             </div>
+             <div class="pagi" tal:condition="primary_category">
+                <a href="/">&lt;<span i18n:translate="Back">Back</span> </a>
```

Comment:
This should link to `/content/list/${primary_category.uuid}` effectively 1 level up from the article.